src/api/initialDataTypes.js

```diff
@@ -294,6 +313,10 @@ export type InitialDataRealm = $ReadOnly<{|
   realm_presence_disabled: boolean,
   realm_private_message_policy: number,
   realm_push_notifications_enabled: boolean,
+
+  // TODO(server-8.0): Added in feat. 231
+  realm_push_notifications_enabled_end_timestamp?: boolean,
```

Comment:
```suggestion
  // TODO(server-8.0): Added in feat. 231
  realm_push_notifications_enabled_end_timestamp?: number | null,
```

since it's "integer | null" in the docs

Reply:
Indeed, thanks for catching!